test/index.test.js

```diff
@@ -0,0 +1,43 @@
+const request = require('supertest');
+const { seedDB, makeReview } = require('../server/seeder.js');
+
+describe('GET /reviews', () => {
+
+  test('Responds with json', (done) => {
+    request('http://localhost:3002')
+      .get('/reviews')
+      .set('Accept', 'application/json')
+      .expect('Content-Type', /json/)
+      .expect(200, done);
+  });
+
+  test('Responds with array of objects', (done) => {
+    request('http://localhost:3002')
+      .get('/reviews')
+      .expect(200)
+      .then((result) => {
+        expect(Array.isArray(result.body)).toBe(true);
+        expect(typeof result.body[0]).toBe('object');
+        done();
+      });
+  });
+});
+
+describe('POST /reviews', () => {
+
+  test('Post responds with updated review', (done) => {
```

Comment:
You can change the test description to POST responds with newly created review. I think you are creating new review and not updating. In case if you implement update make sure to use PUT request.

Reply:
Ah yes. That is correct. It is not an update but a new post.